cmd/draft/config_set.go

```diff
@@ -43,6 +43,9 @@ func (ccmd *configSetCmd) complete(args []string) error {
 }
 
 func (ccmd *configSetCmd) run() error {
+	if globalConfig == nil {
+		return fmt.Errorf("Draft configuration in $DRAFT_HOME/config.toml has not been initialized")
```

Comment:
can we add an action item to the end of the error message? Something like

> Draft configuration in $DRAFT_HOME/config.toml has not been initialized. Run `draft init` to get started

Reply:
Thanks for the feedback, that´s a great idea. I will add that